### ISSUE TITLE
Change in extract-text-webpack-plugin is breaking the vue-loader

### ISSUE THREAD

**Issue by jakubsikora** [vuejs/vue-loader#594]
Latest change to
https://github.com/webpack-contrib/extract-text-webpack-plugin/commit/74b86e0221096c682538472b455118179b7991a3

is breaking https://github.com/vuejs/vue-loader/blob/master/lib/loader.js#L180

Before vue-loader was expecting string, now extract-text plugin brings back an array.

**Comment by mrcsmcln** [comment]
+1

**Comment by jakubsikora** [comment]
```"extract-text-webpack-plugin": "=2.0.0-beta.5",``` as a workaround for now

**Comment by yyx990803** [comment]
Fixed in 10.0.4.

**Comment by zuibunan** [comment]
Not work for me ！

stringifyLoaders may should remove the first item of loaders or it would tip 

> extract-text-webpack-plugin loader is used without the corresponding plugin

```
// stringify an Array of loader objects
function stringifyLoaders (loaders) {
  return loaders.map(function (obj, i) {
    return obj && typeof obj === 'object' && typeof obj.loader === 'string'
      ? obj.loader + (obj.options ? '?' + JSON.stringify(obj.options) : '')
      : obj
  }).slice(1).join('!')
}
```
win10, 
node6.9.5, 
webpack2.2.1,
extract-text-webpack-plugin2.0.0-rc.3

this wasted me a dozen hours

https://github.com/vuejs/vue-loader/commit/c6fc82f7e713d23f84641179692d233f6419cb94#diff-7c7d4918b118d27d18538f53b8155e88R134
